db/migrate/20200326142259_add_job_id_to_report.rb

```diff
@@ -0,0 +1,11 @@
+class AddJobIdToReport < ActiveRecord::Migration[5.2]
+  def up
+    add_column :reports, :job_invocation_id, :integer
+    add_index  :reports, :job_invocation_id
+  end
+
+  def down
+    remove_index  :reports, :job_invocation_id
+    remove_column :reports, :job_invocation_id, :integer
+  end
```

Comment:
This looks like something that should be automatically reversible if you defined just `change` instead of `up` and `down`